mirage/factories/custom-file-metadata-record.ts

```diff
@@ -0,0 +1,24 @@
+import { Factory } from 'ember-cli-mirage';
+import faker from 'faker';
+
+import CustomFileMetadataRecord from 'ember-osf-web/models/custom-file-metadata-record';
+
+
+export default Factory.extend<CustomFileMetadataRecord>({
+    title: faker.lorem.sentence(),
+    description: faker.lorem.sentence(),
+    resource_type_general: 'text',
+    language: 'en-US',
+});
+
+declare module 'ember-cli-mirage/types/registries/model' {
+    export default interface MirageModelRegistry {
+        customFileMetadataRecords: CustomFileMetadataRecord;
```

Comment:
If you look at our other mirage factories (`schema-response`, `review-action`), it seems that for the `MirageModelRegistry` we should do:
```suggestion
        'custom-file-metadata-record': CustomFileMetadataRecord;
```